Corrected broken link for Certificates

cadasi opened the pull request:
We have a link to download a certificate chain that customers need in their Edge Server, but that is broken and pointing to an internal Microsoft Server.
I fixed that pointing to an article with certificate chains that will do the same job.

scanum commented:
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

JohanFreelancer9 commented on Lync/LyncServer/lync-server-2013-migrating-lync-online-users-to-lync-on-premises.md:
````diff
@@ -60,36 +60,8 @@ _**Topic Last Modified:** 2015-11-13_
            ```
 
-2.  Confirm that on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online, as shown in the following table. You can download this chain here: [https://corp.sts.microsoft.com/Onboard/ADFS\_Onboarding\_Pack/corp\_sts\_certs.zip](https://corp.sts.microsoft.com/onboard/adfs_onboarding_pack/corp_sts_certs.zip) .
+2.  Confirm that on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online. You can download this chain here: https://support.office.com/en-ie/article/office-365-certificate-chains-0c03e6b3-e73f-4316-9e2b-bf4091ae96bb
````
```suggestion
2.  Confirm that, on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online. You can download this chain here: https://support.office.com/en-ie/article/office-365-certificate-chains-0c03e6b3-e73f-4316-9e2b-bf4091ae96bb
```

nenonix commented on Lync/LyncServer/lync-server-2013-migrating-lync-online-users-to-lync-on-premises.md:
````diff
@@ -60,36 +60,8 @@ _**Topic Last Modified:** 2015-11-13_
            ```
 
-2.  Confirm that on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online, as shown in the following table. You can download this chain here: [https://corp.sts.microsoft.com/Onboard/ADFS\_Onboarding\_Pack/corp\_sts\_certs.zip](https://corp.sts.microsoft.com/onboard/adfs_onboarding_pack/corp_sts_certs.zip) .
+2.  Confirm that, on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online. You can download this chain here: https://support.office.com/en-ie/article/office-365-certificate-chains-0c03e6b3-e73f-4316-9e2b-bf4091ae96bb
````
```suggestion
2.  Confirm that, on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online. You can download this chain here: https://support.office.com/article/office-365-certificate-chains-0c03e6b3-e73f-4316-9e2b-bf4091ae96bb
```

scanum commented:
@LanaChin  Changes look correct. Could you please review and merge? Thanks!

scanum commented:
@officedocsbot assign @LanaChin

LanaChin commented:
Reviewed changes, added the table back for reference, merged.

cadasi commented:
Hi  Lana,

I had removed the table because it does not represent the reality anymore, so that could create confusion.... In CSS we received questions from customers about not having that certificate chain and having hybrid issues, but that was not was causing their problem :) the certificate chain is better represented in the link I've added

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
From: Lana Chin <notifications@github.com>
Sent: Thursday, May 30, 2019 12:05:56 PM
To: MicrosoftDocs/OfficeDocs-SkypeForBusiness
Cc: Carlos Kamiya; Author
Subject: Re: [MicrosoftDocs/OfficeDocs-SkypeForBusiness] Corrected broken link for Certificates (#2288)


Reviewed changes, added the table back for reference, merged.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F2288%3Femail_source%3Dnotifications%26email_token%3DAJIAB66UG27S5YZ22S7OL3TPYACPJA5CNFSM4HNQRW7KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODWS427Y%23issuecomment-497405311&data=02%7C01%7CCarlos.Kamiya%40microsoft.com%7C4a1fe0171ef14284ddcc08d6e521152d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636948327581437228&sdata=EYTAsy8042N3kQn%2FvHMxSajlfiP%2F1%2Fnu1DajjgghLcU%3D&reserved=0>, or mute the thread<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAJIAB67K3EGETRKMOQ4XD7LPYACPJANCNFSM4HNQRW7A&data=02%7C01%7CCarlos.Kamiya%40microsoft.com%7C4a1fe0171ef14284ddcc08d6e521152d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636948327581447232&sdata=1Y1jiNyn2itAgBbkzJcJkJvssvI5w9KkNSqN8XNm8JU%3D&reserved=0>.
